django/forms/widgets.py

```diff
@@ -787,6 +787,19 @@ class CheckboxSelectMultiple(RendererMixin, SelectMultiple):
     renderer = CheckboxFieldRenderer
     _empty_value = []
 
+    def build_attrs(self, extra_attrs=None, **kwargs):
+        attrs = super(CheckboxSelectMultiple, self).build_attrs(extra_attrs, **kwargs)
+        # The required attribute comes from the bound field if the form field
+        # is required. When required is set on multiple checkboxes with the
+        # same name, the browser will require each checkbox be checked to
+        # submit the form. This is not the desired validation, so remove the
+        # attribute.
+        try:
+            del attrs['required']
+        except KeyError:
+            pass
```

Comment:
```python
if 'required' in attrs:
  del attrs['required']
```
or
`attrs.pop('required', None)` might be better here unless I am missing something and you want to handle `KeyError` specifically.